weave/trace/weave_client.py

```diff
@@ -1976,6 +1995,132 @@ def send_table_create() -> TableCreateRes:
 
         return table_ref
 
+    def _should_use_chunking(self, table: Table | WeaveTable) -> tuple[bool, bool]:
```

Comment:
nit: don't return a tuple, it's hard to understand.  Return a dataclass or at least a typeddict